File: ts/src/BluefinShieldconexError.ts

import { Context } from './Context'


class BluefinShieldconexError extends Error {

  isBluefinShieldconexError = true

  sdk = 'BluefinShieldconex'

  code: string
  ctx: Context

  constructor(code: string, msg: string, ctx: Context) {
    super(msg)
    this.code = code
    this.ctx = ctx
  }

}

export {
  BluefinShieldconexError
}

File: dart/lib/BluefinShieldconexError.dart
class BluefinShieldconexError extends Error {
  final bool isBluefinShieldconexError = true;

  final String sdk = 'BluefinShieldconex';

  String code;
  String message;
  dynamic ctx;

  // Populated by makeError with the (cleaned) result and spec.
  dynamic result;
  dynamic spec;

  BluefinShieldconexError(this.code, this.message, [this.ctx]);

  @override
  String toString() => 'BluefinShieldconexError: ' + code + ': ' + message;
}
